dev-util/envman/envman-2.2.8.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Please use repoman to commit or at least fix this manually to say `-2020 Gentoo Authors`

Reply:
ok